admin/src/components/DeleteEntity.js

```diff
@@ -0,0 +1,50 @@
+import { Button, Modal } from "react-bootstrap";
+import { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const DeleteEntity = ({
```

Comment:
Ez jó ötlet volt, viszont egy fokkal lehet még általánosabb: propsban nem is kell az isMarket, csak marketName, vendorName helyett mondjuk az egész question, handleDeleteMarket, handleDeleteVendor helyett handleDelete, marketId, vendorId helyett csak id. És akkor kívülről a használat helyén meg ügyesen fel lehet paraméterezni. (Így akár user törlésre is jó lesz később.)